modules/mockk-dsl/src/commonMain/kotlin/io/mockk/API.kt

```diff
@@ -26,7 +26,7 @@ object MockKDsl {
     inline fun <reified T : Any> internalMockk(
         name: String? = null,
         relaxed: Boolean = false,
-        vararg moreInterfaces: KClass<*>,
+        moreInterfaces: Array<out KClass<*>>,
```

Comment:
I was not sure whether to use `List` or `Array`, but I chose `Array` for the following reasons.

- `MockKGateway` uses `Array` as an argument in the same situation.
- It is more efficient not to convert to `List`.
- It is an `internal` API, and it seems acceptable to sacrifice some readability.